### PR TITLE
Update description and tags in readme

### LINKED ISSUE #3449: Improve plugin description to indicate official status
## Feature description

There are multiple “AMP” plugins in the ecosystem. Users need to be able to easily identify which plugin is officially supported and maintained and by AMP project contributors. The official status is indicated in the plugin's banner and description:

> image

However, it does not indicate “Official AMP plugin for WordPress” when looking at the plugin directory list:

Screen Shot 2019-10-04 at 14 58 20

Also, there should be some mention of “Accelerated Mobile Pages” somewhere, that this is what it used to stand for.

```diff
diff --git a/readme.txt b/readme.txt
index e9d4ec3c..a1726ac6 100644
--- a/readme.txt
+++ b/readme.txt
@@ -8,11 +8,11 @@ License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Requires PHP: 5.4
 
-Enable AMP on your WordPress site, the WordPress way.
+Official AMP plugin WordPress, supported and maintained by AMP project contributors. Enable AMP on your WordPress site, the WordPress way. Formerly known as “Accelerated Mobile Pages”.
 
 == Description ==
 
-The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
+The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. AMP used to stand for “Accelerated Mobile Pages” but no longer as it is for improving beb experiences regardless of the device type. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
 
 Features and capabilities provided by the plugin include:
```

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by westonruter**
## Summary

Fixes #3449

### Before

> image

> Screen Shot 2019-10-10 at 14 42 24

### After

> image

> image

## Checklist

- [x] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # AMP Plugin for WordPress
 
 ![Banner](wp-assets/banner-1544x500.png)
-Enable AMP on your WordPress site, the WordPress way.
+Official AMP plugin, supported and maintained by AMP. Formerly “Accelerated Mobile Pages”, now for mobile and desktop. Use for website and stories!
```
Comment (mattludwig) [inline readme.md]: What about... "Official AMP plugin, supported by the AMP team. Formerly "Accelerated Mobile Pages", AMP enables great experiences across mobile, desktop & stories."  ?

File: readme.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Enable AMP on your WordPress site, the WordPress way.
 
 ## Description ##
 
-The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
+The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. AMP used to stand for “Accelerated Mobile Pages” but no longer as it is for improving web experiences regardless of the device type, whether mobile or desktop. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
```
Comment (mattludwig) [inline readme.md]: Change "AMP used to stand for “Accelerated Mobile Pages” but no longer as it is for improving web experiences regardless of the device type, whether mobile or desktop." to --> "AMP formerly stood for "Accelerated Mobile Pages", but now works completely across desktop and mobile."

**Comment by westonruter** [comment]
Thanks! Pushed up changes directly to WordPress.org: https://plugins.trac.wordpress.org/changeset/2172612/

**Comment by westonruter** [comment]
I had to remove the quotes and the ampersand, because in the former the curly quotes counted for more than 1 byte due to UTF-8 encoding, and in the latter because `&` was encoded as `&amp;` thus counting for 5 characters, causing it to be cut off:

image

**Comment by swissspidy** [comment]
Merging since this is live already.

**Pull request merged**